docs/reference/sciencedirect/ObjectMetadata.rst

```diff
@@ -0,0 +1,40 @@
+pybliometrics.sciencedirect.ObjectMetadata
+==========================================
+
+All components of a Document that are not text (figures, formulas, etc.) are called `objects`.
+`ObjectMetadata()` retrieves the metadata associated with all objects in a document.
+Specifically, it retrieves the metadata from the `ScienceDirect Object Retrieval API <https://dev.elsevier.com/documentation/ObjectRetrievalAPI.wadl>`_.
+
+.. currentmodule:: pybliometrics.sciencedirect 
+.. contents:: Table of Contents
+    :local:
+
+Documentation
+-------------
+
+.. autoclass:: ObjectMetadata
+   :members:
+   :inherited-members:
+
+Examples
```

Comment:
Please show how the results can be used to generate a pandas DataFrame. Have a look at the ScopusSearch() class